api/src/utils/get-cache-key.ts

```diff
@@ -6,11 +6,12 @@ import { pick } from 'lodash';
 export function getCacheKey(req: Request): string {
 	const path = url.parse(req.originalUrl).pathname;
 	const isGraphQl = path?.includes('/graphql');
+	const isGet = req.method === 'GET';
```

Comment:
For consistency with other usages of `req.method`.

```suggestion
	const isGet = req.method.toLowerCase() === 'get';
```

Reply:
Fixed 👍